sys/include/riotboot/usb_dfu.h

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (C) 2020 Mesotic SAS
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    sys_riotboot_usb_dfu   Initialization of USB Device Firmware
+ *                                     Upgrade for riotboot
+ * @ingroup     sys
+ * @{
+ *
+ * @file
+ * @brief       USB DFU initialization for riotboot
+ *
+ * @author      Dylan Laduranty <dylan.laduranty@mesotic.com>
+ *
+ * @}
+ */
+
+#ifndef RIOTBOOT_USB_DFU_H
+#define RIOTBOOT_USB_DFU_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name USB DFU RAM information for riotboot
+ * @{
+ */
+#ifndef RIOTBOOT_DFU_ADDR
+#define RIOTBOOT_DFU_ADDR      (CPU_RAM_BASE + CPU_RAM_SIZE - 4)   /**< DFU default magic address */
+#endif
+#ifndef RIOTBOOT_MAGIC_NUMBER
+#define RIOTBOOT_MAGIC_NUMBER  (0xDEADBEEFUL)                      /**< DFU default magic value */
```

Comment:
Why not reuse the existing value from `sys/include/riotboot/hdr.h` and keep the slots compatible across bootloader versions

Reply:
Changed.